api/user/userModel.js

```diff
@@ -28,6 +28,13 @@ const findUserData = async (role_id, profile_id) => {
   }
```

Comment:
Looks like the line 12 statement and line 20 statement is exactly the same. Could we change the line 12 one because that should be for the role of admin, not parents

Reply:
Hi @sagun1151, just changed line 12 to the role admin as per your request